scoring/performance_profile.py

```diff
@@ -25,15 +25,16 @@
   The keys in this dictionary should match the workload identifiers used in
   the dictionary of submissions.
 """
-
 import itertools
 import operator
 import os
 import re
 
+from absl import logging
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
+import scoring_utils
```

Comment:
I think we use absolute paths everywhere else.

Reply:
yes you're right, fixed